### DISCUSSION TITLE
Percentage on aroma diffuser

### DISCUSSION THREAD

**Discussion by teeodor** [make-all/tuya-local#1587] [Q&A]
Hi, I have a problem with a tuya aroma diffuser.
It has 2 fan mode, minimum or maximum.
For minimum just set percentage at 50%, for maximum percentage it’s “big”.
I’m stuck with no ideas for set the word big in percentage

**Comment by make-all** [comment]
Is this using a predefined config, or are you writing your own?  If you are using a predefined one, it looks like your device is not 100% compatible with the config, as the device is using different strings for the fan speed.

**Reply by teeodor** [reply]
No, I have used default config

**Reply by make-all** [reply]
There is no "default config".  There are over 600 different configs in this integration, at least 9 of which are for aroma diffusers.  You need to give more details about your specific situation up front if you want help.

**Reply by teeodor** [reply]
Sorry but I don't understand.
I add the device by the integration, and I have configured nothing.
In the debug I read type: ditua_dt1522yn_aromadiffuser

**Reply by make-all** [reply]
Is your device a Ditua DT1522YN aroma diffuser?  Or something else?

If it is something else, then you may have to file a request for a new device.

**Reply by teeodor** [reply]
https://maxcio.com/products/smart-aroma-diffuser-brown
This is it

**Reply by teeodor** [reply]
I buy it 2 or 3 years ago
